src/resources/breaking-changes.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Breaking changes and deprecations
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists all language and library breaking changes and deprecations in Dart,
+organized by release and area, to help Dart users understand and manage their
+impact. Complete release notes are available in the [Dart SDK changelog][changelog].
+The [breaking change policy][] document describes the policy and process
+around breaking changes and deprecations in Dart. 
+
+**This page includes two types of breaking changes, as well as deprecations**:
+* **Unversioned**: The compiler does not maintain backward compatibility, and
+  code may break as soon as you [upgrade your sdk version][sdk] if it relies on
+  the previous behavior.
+  
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: The compiler maintains backward compatibility for
+  existing code, and the behavior change only takes effect (potentially breaking
+  code that relies on the previous behavior) when you upgrade the
+  [language version] of your code.
+  
+  *These are marked "**Language versioned**"*.
+* **Deprecations**: The compiler maintains compatibility for deprecated code,
+  with a warning. Deprecations are then completely removed in the release
+  following their deprecation, breaking any code that relies on the previous behavior.
+  
+  *These are marked "**Deprecated**" and "**Removed**", respectively*.
+
+If you have questions or concerns about any of these breaking changes, please 
+comment on the breaking change issue linked from the relevant entry.
+To be notified about future breaking changes, join the [Dart announce][] group.
+
+[breaking change policy]: https://github.com/dart-lang/sdk/blob/main/docs/process/breaking-changes.md
+[changelog]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md
+[sdk]: /get-dart
+[language version]: /guides/language/evolution#language-versioning
+[Dart announce]: https://groups.google.com/a/dartlang.org/g/announce
```

Comment:
I feel like we have a few places we link to this now. Consider adding a site variable in the [`config.yml`](https://github.com/dart-lang/site-www/blob/main/_config.yml#L32-L40).

Reply:
So how does this work? When I add the link to config.yml does that mean I no longer need to include this line? I can just write `[Dart announce]` anywhere on the site and it will link?

Reply:
You'll instead just link to `{{site.announce}}` (or whatever you name it) so you don't have to find/time that long URL everytime. Plus if it ever changes, it'll be easy to update.

```
[Dart announce]: {{site.announce}}
```